eng/Versions.props

```diff
@@ -28,7 +28,7 @@
   <PropertyGroup>
     <!-- Versions used by several individual references below -->
     <RoslynDiagnosticsNugetPackageVersion>2.6.2-beta2</RoslynDiagnosticsNugetPackageVersion>
-    <CodeStyleLayerCodeAnalysisVersion>2.8.2</CodeStyleLayerCodeAnalysisVersion>
+    <CodeStyleLayerCodeAnalysisVersion>2.9.0</CodeStyleLayerCodeAnalysisVersion>
```

Comment:
📝 This is the version where `SyntaxToken` is marked `readonly`. The FormattingAnalyzer NuGet package will only work with 15.8+ after this change.

Reply:
Use of `in SyntaxToken` in the formatter has been removed from this pull request (will be submitted separately).